lib/LaTeXML/Package/xylatexml.tex.ltxml

```diff
@@ -0,0 +1,994 @@
+  beforeConstruct => \&xy_FillStroke,
+  properties      => sub {
+    my ($x0, $y0, $x1, $y1) = map { LookupRegister($_); } '\X@p', '\Y@p', '\X@c', '\Y@c';
+    my $l = LookupRegister('\xybsqll@');
+    # Note: We perversely use the pattern to control dashes
+    my $dashed = ToString(Digest(T_CS('\lx@xy@pattern')));
+    my $w      = $x1->subtract($x0);
+    my $h      = $y1->subtract($y0);
+    my $r      = $l->multiply(0.66);
+    my $n      = int((0.5 + sqrt($w->valueOf * $w->valueOf + $h->valueOf * $h->valueOf)) / $l->valueOf);
+    $n++    if ($n % 1);
+    $n += 2 if $dashed && !($n % 4);    # make odd
+    my $dx   = $w->divide($n);
+    my $dy   = $h->divide($n);
+    my @path = ();
+    my ($x, $y) = ($x0, $y0);
+    # NOTE: Work out how to use 'A' path steps to make it smooth
+    my $step  = ($dashed ? 4 : 2);
+    my $xstep = $dx->multiply($step);
+    my $ystep = $dy->multiply($step);
+    for (my $i = 0 ; $i < $n ; $i += $step) {
+      push(@path, 'M', $x, $y) if $dashed || ($i == 0);
+      push(@path, 'a', $r, $r, 60, 0, 0, $dx, $dy);
+      push(@path, 'a', $r, $r, 60, 0, 1, $dx, $dy);
+      $x = $x->add($xstep); $y = $y->add($ystep); }
+    #    Debug("SQUIGGLE $x0, $y0 to $x1, $y1 in $n parts: " . join(' ', @path)) if $LaTeXML::DEBUG{xy};
+    (path => xy_packpath(@path),
+      width     => Dimension(0),             height  => Dimension(0), depth => Dimension(0),
+      do_stroke => LookupValue('xy_stroke'), do_fill => 0); });
+
+#======================================================================
+# Tips
+
+# Redefinitions:
+
+# These badsically ammount to replacements for the glyphs in xyatipfont, xybtipfont
+DefMacro('\atip@@',       '\lx@xy@tip{1}');                                # upper part of ">" tip
+DefMacro('\btip@@',       '\lx@xy@tip{-1}');                               # lower part of ">" tip
+DefMacro('\Tip@@',        '\lx@xy@tip{1.5}\lx@xy@tip{-1.5}');              # large tip
+DefMacro('\Ttip@@',       '\lx@xy@tip{2}\lx@xy@tip{-2}');                  # extra large tip
+DefMacro('\stopper@@',    '\lx@xy@stopper');                               # "|" tip
+DefMacro('\hook@@',       '\lx@xy@hook{0}');                               # "(" tip
+DefMacro('\ahook@@',      '\lx@xy@hook{1}');                               # "(" tip shifed up
+DefMacro('\bhook@@',      '\lx@xy@hook{-1}');                              # "(" tip shifted down
+DefMacro('\aturn@@',      '\lx@xy@turn{1}');                               # quarter circle tip  up
+DefMacro('\bturn@@',      '\lx@xy@turn{-1}');                              # quarter circle tip down
+DefMacro('\solidpoint@',  '\pointlike@{\lx@xy@fill@on\lx@xy@point}\jot');  # filled circle
+DefMacro('\hollowpoint@', '\pointlike@{\lx@xy@fill@off\lx@xy@point}\jot'); # empty circle
+
+# Redefined to position w/o kerns.
+RawTeX(<<'EoTeX');
+%\xydef@\Tip@{\kern2.5pt \vrule height2.5pt depth2.5pt width\z@
+% \Tip@@ \kern2.5pt \egroup
```

Comment:
Generally, could we take out the `%` comment lines out of the RawTeX and into a perl `#` comment?

Saves on runtime, but more importantly it also makes the RawTeX itself easier to review, as I don't have to double-guess typos.